.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -17,6 +17,7 @@ trigger:
       - LICENSE-3rdparty.csv
       - NOTICE
       - .azure-pipelines/noop-pipeline.yml
+      - .vsconfig
```

Comment:
can you add it to the `.github/workflows/profiler-pipeline.yml` file please ?

Reply:
This file did not exist on this branch yet, so I merged `master` in and made the change.

Reply:
lol, do `.azure-pipelines/ultimate-pipeline.yml` and `.github/workflows/profiler-pipeline.yml` need to ignore each other as well?